service/templates/admin.html

```diff
@@ -707,11 +716,14 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										  <md-radio-button value="">None</md-radio-button>
 									</md-radio-group>
 								</md-card-content>
+								<md-input-container class="control" style="width:45%;">
```

Comment:
This is flush with the left side of the card, you probably want to add a bit of padding to it

Reply:
I fixed it by putting a `<br>`.